src/ElectronApp.ts

```diff
@@ -1,41 +1,47 @@
-import { WEB_DIR, EVENTS, environment, preferences, EventBus } from 'remoteit-headless'
+import { WEB_DIR, EVENTS, environment, preferences, EventBus, Logger } from 'remoteit-headless'
+import AutoUpdater from './AutoUpdater'
 import electron from 'electron'
 import TrayMenu from './TrayMenu'
-import AutoUpdater from './AutoUpdater'
 import debug from 'debug'
 import path from 'path'
 import url from 'url'
 
 const d = debug('r3:headless:ElectronApp')
 
+const DEEP_LINK_PROTOCOL = 'remoteit'
+const DEEP_LINK_PROTOCOL_DEV = 'remoteitdev'
```

Comment:
Needed to have different protocols for dev and production, because production tests were opening electron!